Bottom: 796e3c7c3f354e37bd02512a2643cc632c0d5a09
Top:    2c16c7a67257752e1ab917b24a8eb4bd307e6711
Author: Igor Stoppa <igor.stoppa@huawei.com>
Date:   2018-06-07 16:06:51 +0300

Use designated initializer for struct list_head in list.h


---

diff --git a/include/linux/list.h b/include/linux/list.h
index 4b129df4d46b..11847c96571f 100644
--- a/include/linux/list.h
+++ b/include/linux/list.h
@@ -18,7 +18,10 @@
  * using the generic single-entry routines.
  */
 
-#define LIST_HEAD_INIT(name) { &(name), &(name) }
+#define LIST_HEAD_INIT(name) {	\
+	.next = &(name),	\
+	.prev = &(name),	\
+}
 
 #define LIST_HEAD(name) \
 	struct list_head name = LIST_HEAD_INIT(name)
